### DISCUSSION TITLE
what does the strength_clip of LoraLoader mean?

### DISCUSSION THREAD

**Discussion by Gasskin** [comfyanonymous/ComfyUI#215] [Q&A] [answered]
have tried for a white,but no regular find...
need i always set it to 1?

**Comment by comfyanonymous** [comment, marked as answer]
If you don't know set them to the same value but I let you set them separately because of:
https://github.com/cloneofsimo/lora#what-happens-to-text-encoder-lora-and-unet-lora

**Reply by Gasskin** [reply]
thanks,it help me a lot

**Reply by CyrilCommando** [reply]
> If you don't know set them to the same value but I let you set them separately because of: https://github.com/cloneofsimo/lora#what-happens-to-text-encoder-lora-and-unet-lora

Could you maybe try to... teach us.. something? I know it might be tough, but at least it would be of value.
